### ISSUE TITLE
UI does surrounding search when open log item

### ISSUE THREAD

**Issue by sega-yarkin** [Graylog2/graylog2-server#2766]
I've updated to the last stable release. When I open a search result and click to any log line, UI opens it and at the same time navigates to surrounding search with this log item (+- 5 minutes).
I've cleared browser chache but nothing changed.

## Expected Behavior
Just open log item info.

## Current Behavior
Open log info and navigate to surrounding search.

## Your Environment

* Graylog Version: 2.1.0+62db7e0
* Browser version: Google Chrome 52

**Comment by jalogisch** [comment]
@sega-yarkin 

I tried to reproduce this behaviour but I was not able to. Could you, please try again and give some more details. Maybe with Screenshots that shows where you click. 

Thank you

**Comment by sega-yarkin** [comment]
@jalogisch 
Sorry for disturbing, it was my fault, but it looks weird.
Before the stable release I've tried to update to last beta but haven't found deb-package and I just unpacked from archive and replaced the main jar file and plugins' jar files. Last update to stable release replaced the main jar, but not plugins' files because they have suffix "-beta" and there were two copies of every plugins "stable" and "beta". I've deleted beta versions of plugins, restarted Graylog server and everything is fine now. Thanks)
